pkg/machine/applehv/machine.go

```diff
@@ -946,15 +946,17 @@ func (m *MacMachine) forwardSocketPath() (*machine.VMFile, error) {
 	return machine.NewMachineFile(filepath.Join(path, sockName), &sockName)
 }
 
+// resizeDisk uses os truncate to resize (only larger) a raw disk.  the input size
+// is assumed GB
 func (m *MacMachine) resizeDisk(newSize uint64) error {
-	// TODO truncating is not enough; we may not be able to support resizing with raw image?
-	// Leaving for now but returning an unimplemented error
-
-	//if newSize < m.DiskSize {
-	//	return fmt.Errorf("invalid disk size %d: new disk must be larger than %dGB", newSize, m.DiskSize)
-	//}
-	//return os.Truncate(m.ImagePath.GetPath(), int64(newSize))
-	return define.ErrNotImplemented
+	if newSize < m.DiskSize {
+		return fmt.Errorf("invalid disk size %d: new disk must be larger than %dGB", newSize, m.DiskSize)
+	}
+	newByteSize, err := units.RAMInBytes(fmt.Sprintf("%dGB", m.DiskSize))
+	if err != nil {
+		return err
+	}
```

Comment:
wouldn't it be much cheaper to just multiply by 10^9 or assuming this value is actually meant to be GiB then just 2^30 which would be shifting 30 bytes so `newSize<<30`